release-image/bootstrap.sh

```diff
@@ -91,6 +91,20 @@ case "$cmd" in
       docker manifest push aztecprotocol/aztec:$(dist_tag)
     fi
     ;;
+  "push-pr")
+    echo_header "release-image push-pr"
```

Comment:
nit: Since this doesn't push to a pull-request, maybe `push-pr-build`. Also more idiomatic (for ci3) is `push_pr_build`